.gitignore

```diff
@@ -1,4 +1,5 @@
 # Miscellaneous
+*.exe
```

Comment:
```suggestion
```

Reply:
> I'll leave this to someone else to review, as CRS really isn't my strength.

No worries, though there's zero math involved. This change is mostly just moving things around to avoid overhad.

>  I did notice from your on-device comparison that the Raster times increased significantly? I'm assuming that was a coincidence, as these changes shouldn't place any more strain on the raster thread.

That's probably accidental. The raster times shouldn't change at all.

I did continue my explorations. I've tried for example vector operations with Float64x2 and single-precision floats with Float32x4. None of them did too much, happy to share my benchmark results if there's more interest. However, I did find two more worthwhile optimizations: 
 * avoid Num.clamp
 * monomorphize the common EPSG3857/spherical mercator case.

This saved another 22% cpu time. Which if you compare HEAD to this PR is a ~41% reduction for my polygon testcase (your mileage may vary). You can also see that the raster thread remains unchanged.

![reduction](https://github.com/fleaflet/flutter_map/assets/111986/6c0289cf-1b50-4bb6-a99e-350d580e6dad)

Reply:
Wow, I'm surprised that `clamp` is that inefficient, I'll have to remember that.

Reply:
In absolute terms it's not super slow, it's just that clamping is such a cheap operation that the virtual function overhead is large in relative terms, especially when you call it in a tight loop.